test/unit/Validation/Constraint/ValidAtTest.php

```diff
@@ -38,6 +38,7 @@ public function createDependencies(): void
      *
      * @uses \Lcobucci\JWT\Token\DataSet
      * @uses \Lcobucci\JWT\Token\Plain
+     * @uses \Lcobucci\JWT\Token\Signature
```

Comment:
Just noticed all these `@uses`: since I didn't notice before, what is the conveyed intent?